core/account/indexer.go

```diff
@@ -51,8 +51,14 @@ func (m *Manager) indexAnnotatedAccount(ctx context.Context, a *Account) error {
 	})
 }
 
-type output struct {
+type outputWithOutpoint struct {
```

Comment:
Good call. Renamed to `rawOutput` and `accountOutput`: 31e2a038b99d51df50b6c26d292a1db49e72d7c2